pkg/system/filesys_windows.go

```diff
@@ -0,0 +1,62 @@
+// +build windows
+
+package system
+
+import (
+	"os"
+	"regexp"
+	"syscall"
+)
+
+// Volume Path aware implementation of MkdirAll for Windows.
+func MkdirAll(path string, perm os.FileMode) error {
+	re := regexp.MustCompile("^\\\\\\\\\\?\\\\Volume{[a-z0-9-]+}$")
```

Comment:
Aren't we better off using [`raw string literals`](https://golang.org/ref/spec#String_literals) for this kind of things?